fk/teamapply.go

```diff
@@ -69,8 +69,23 @@ func teamApply(teamUUID uuid.UUID) exitCode {
 		"This information allows them to verify your request.",
 	}))
 
-	out.Print("Your team admin will need to authorize your request for Fluidkeys to\n" +
-		"start working.\n\n")
+	out.Print(formatFileDivider("Please authorize me to join Kiffix", 80) + "\n")
+	requestMessage := `I've requested to join ` + teamName + ` on Fluidkeys.
```

Comment:
i think we probably shouldn't use the raw string operator in the middle of code like this as it makes it quite hard to see the structure of indents. Nothing wrong with "...\n" I reckon

Reply:
I've added a commit for this.